content/fr/real_user_monitoring/browser/advanced_configuration.md

```diff
@@ -177,6 +177,36 @@ window.DD_RUM &&
 
 **Remarque** : respectez la [convention de nommage Datadog][2] pour améliorer la corrélation de vos données sur l'ensemble de la solution.
 
+### Lire le contexte global
```

Comment:
afaik, no need to update the translation yourself, it will be handle by translators afterwards

Reply:
This is correct. If these files haven't been translated by a native speaker, best to remove so we can put them through the translation process.

Reply:
ok, I removed the japanese version.